platform/darwin/src/MGLAccountManager_Private.h

```diff
@@ -16,7 +16,6 @@ NS_ASSUME_NONNULL_BEGIN
 #if TARGET_OS_IPHONE || TARGET_OS_SIMULATOR
 /// The current global SKU.
 @property (class, atomic, readonly, nullable) NSString *skuToken;
```

Comment:
I missed removing this `nullable` — will do soon.

Reply:
→ https://github.com/mapbox/mapbox-gl-native/pull/14578/commits/a7b2d9e0c6315ac65443e71fc806922dd536c5b5